packages/scripts/src/sdkGenerator.ts

```diff
@@ -97,7 +97,7 @@ export class MethodGenerator extends Generator<Models.IApiModel> {
     const items: string[] = []
     // reset refcounts for ALL types so dynamic import statement will work
     Object.entries(this.model.types).forEach(([_, type]) => (type.refCount = 0))
```

Comment:
This caught my eye. What do you think of replacing `Object.entries` with `Object.values` here to avoid the slightly less easy on the eyes destructuring syntax? 
```suggestion
    Object.values(this.model.types).forEach(type => type.refCount = 0)
```

Reply:
This loop is going to need to change to iterate sorted collections in the right order